api/commands/reminders/remind.js

```diff
@@ -41,34 +39,52 @@ module.exports = class RemindCommand extends Command {
   }
 
   run(msg, { target, content, datetime }) {
-    const author = {
-      username: msg.message.author.username,
-      discriminator: msg.message.author.discriminator
-    };
+    const { author } = msg.message;
 
-    let parsedTime;
+    let timezone;
+    let reminders = [];
+    let remindersJson;
 
     if (!chrono.parseDate(datetime)) {
       return new Promise((resolve, reject) =>
         reject(msg.say(exceptions.invalid_datetime_format)));
     }
 
-    return selectTz([author.username, author.discriminator])
-    .then((timezone) => {
-      if (!timezone) {
-        return Promise.reject(msg.say(exceptions.timezone_not_set));
+    return tzStore.getAsync(author.id)
+    .then((tz) => {
+      if (!tz) {
```

Comment:
should this be a guard clause? looks like itll continue through even if the tz is not set -- something that fires the msg but breaks the chain
similar to line 64-66 (https://github.com/jnhallquist/discord-reminders-bot/pull/48/files#diff-2fdfa5921b259f4983e0da82ee856f67R64)
```
if (!tz) return Promise.reject(msg.say(exceptions.timezone_not_set));
timezone = tz;
```